src/CoursePageResources/CoursePageResources.jsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { injectIntl } from '@edx/frontend-platform/i18n';
+import { Button } from '@edx/paragon';
+import CoursePage from '../course-page/CoursePage';
+
+function CoursePageResources() {
+  return (
+    <main>
+      <div className="container-fluid">
+        <div className="container-fluid-container d-flex justify-content-between align-items-center border-bottom">
+          <h1 className="container-fluid-heading mt-3">Pages & Resources</h1>
+          <Button className="container-fluid-button font-weight-bold rounded my-3">
+            View Live
+          </Button>
```

Comment:
Using the following, I get a rounded button instead of the rectangular button in the wireframe. What should I do?
```
          <a class="btn btn-primary" href="#" role="button">                                                             
            {intl.formatMessage(messages['course-authoring.pages-resources.viewLive.button'])}                           
          </a>                                                                                                           
```

Reply:
@taniwha Feel free to just leave it as is for right now.
We'll have the UI folks run over the details later to confirm expectations.